src/components/collect-button.tsx

```diff
@@ -6,8 +6,10 @@ import "./collect-button.sass";
 import { Mouse, MouseType, UNCOLLECTED_IMAGE } from "../models/mouse";
 
 interface IProps extends IBaseProps {
-  mouse?: MouseType;
-  index: number;
+  backpackIndex?: number;
```

Comment:
I made a handful of changes to `collect-button` - first, I removed the `mouse` from the props, since the `backpackIndex` alone is enough to determine the mouse. The more substantial changes were to allow me to reuse the buttons in the cell zoom UI. I want the buttons to look exactly the same, but the behavior is quite different so I added click handler overrides.

In the end I'm not sure it was worth adding these complications (e.g. allowing for an undefined `backpackIndex`), but I wanted to avoid copy/pasting the entire file. Let me know if you have any suggestions.

Reply:
I think this makes sense. I think a clearer solution *might* have been to make an `OrganismCircle` component which handles the image and has generic click handlers, and then `BackpackItem` component would be a very thin wrapper on top of that, and the `CellZoom` would bring it in directly. But I think that this is fine.

Reply:
^ agreed, that would look nicer, I'll give it a try if I have a chance to do some cleanup